tests/console/test_console_availability.py

```diff
@@ -0,0 +1,50 @@
+import getpass
+import pexpect
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+@pytest.mark.parametrize("target_line", ["1", "2", "3", "4"])
+def test_console_availability(duthost, creds, target_line):
+    """
+    Test console are well functional.
+    Verify console access is available after connecting from DUT
+    """
+    hostip, hostuser = "172.17.0.1", getpass.getuser()
```

Comment:
Seems like this test will only available on vtestbed. Can we add pytestmark to limit it KVM test only?
Is possible to get the host ip from our configuration file?

Reply:
1. Pytestmark has been added
2. `172.17.0.1` is the default gateway ip in docker bridge network, and it's static. So I think it's unnecessary to get it from config file.